assets/scripts/streets/StreetName.jsx

```diff
@@ -11,9 +11,10 @@ class StreetName extends React.PureComponent {
     id: PropTypes.string,
     name: PropTypes.string,
     childRef: PropTypes.func,
-    onClick: PropTypes.func,
+    handleStreetNameChange: PropTypes.func,
     isStreetReadOnly: PropTypes.bool,
-    isHoverable: PropTypes.bool
+    isHoverable: PropTypes.bool,
+    editable: PropTypes.bool
```

Comment:
The `editable` prop reads from the same value as the `isStreetReadOnly` prop. (See the `mapDispatchToProps` function at the bottom of both `StreetNameCanvas` and `StreetName` components.) We should use one or the other. (I do prefer using the `editable` prop passed in by the parent component, and we should set the default value of `editable` to false.)